plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
```

Comment:
5 minutes seems pretty short for this for a zone refresh.  Also does our public-facing authoritative servers accept zone change notifications from the shadow master?  If so, then such a low value is even less necessary.